data/json/construction.json

```diff
@@ -141,6 +141,18 @@
     "pre_terrain": "t_door_frame",
     "post_terrain": "t_door_c"
   },
+  {
+    "type": "construction",
+    "description": "Fill Shallow Water With Dirt",
+    "//": "Fills a shallow water with dirt.",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "survival", 1 ] ],
+    "time": 60,
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "components": [ [ [ "material_soil", 500 ] ] ],
```

Comment:
A similar amount of sand or gravel would probably work as options too.

Refresh my memory but is this the amount of dirt from a shallow or a deep pit?

Reply:
This is 500 x 5L portions - or 1250 liters of dirt. This is 10x the amount from a shallow pit, and is a bit more than a deep pit (which is about 400  portions, or 1000L)

Reply:
I cant see that this has been addressed. What do you think the amount should be? and is it preferable that sand and gravel are options in the components? would that be ok that it still creates t_dirt?

Reply:
I would say the amount should be the same or just slightly higher than a shallow pit. 50-60. And yeah an equivalent amount of sand or gravel should be listed as options although that's not as vital

Reply:
I don't think sand or gravel would result in a dirt tile, so maybe it's not a good idea to complicate my first contribution like this.